deploy/Dockerfile

```diff
@@ -118,6 +118,12 @@ RUN mkdir -p /composer && \
 COPY composer.* /composer/
 RUN cd /composer && composer install
 
+# Setup Vault
+RUN wget https://releases.hashicorp.com/vault/1.0.2/vault_1.0.2_linux_amd64.zip && \
```

Comment:
It's better to separate the version in a separate variable. 

See for example - [separate version](https://github.com/rtCamp/dockerfiles/blob/5d1eb9465caf12b36e8529ad4c018691b16271e9/ci-deployer/Dockerfile#L13) and then [use it in the install command](https://github.com/rtCamp/dockerfiles/blob/5d1eb9465caf12b36e8529ad4c018691b16271e9/ci-deployer/Dockerfile#L70)

Reply:
Updated to use env variable.